subsys/bluetooth/host/iso.c

```diff
@@ -36,7 +36,8 @@ LOG_MODULE_REGISTER(bt_iso);
 
 #if defined(CONFIG_BT_ISO_UNICAST) || defined(CONFIG_BT_ISO_SYNC_RECEIVER)
 NET_BUF_POOL_FIXED_DEFINE(iso_rx_pool, CONFIG_BT_ISO_RX_BUF_COUNT,
-			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_RX_MTU), 8, NULL);
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_RX_MTU),
+			  sizeof(struct bt_buf_data), NULL);
```

Comment:
Should this be 
```suggestion
			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_RX_MTU),
			  sizeof(struct iso_data), NULL);
```

We seemingly have this macro 
```c
#define iso(buf) ((struct iso_data *)net_buf_user_data(buf))
```

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/host/iso.c#L579

Reply:
For me it is unclear when/where `struct bt_buf_data` is used. Is it ever used for ISO? Otherwise I suggest to just have `struct iso_data` here.

Reply:
> For me it is unclear when/where `struct bt_buf_data` is used. Is it ever used for ISO? Otherwise I suggest to just have `struct iso_data` here.

It's used here(https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/host/iso.c#L579) at first, and then used for ISO.

Reply:
```C
struct iso_data {
	/** BT_BUF_ISO_IN */
	uint8_t  type;

	/* Index into the bt_conn storage array */
	uint8_t  index;

	/** ISO connection handle */
	uint16_t handle;
};
```

The `struct iso_data` extend (implicit) `struct bt_buf_data`, but not as intuitive as `struct acl_data` (https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/host/conn_internal.h#L151)
I don't want to revise it in depth, so I'll accept your proposal and just keep the first one.